resources/Dockerfile

```diff
@@ -0,0 +1,2 @@
+FROM nginx:alpine
```

Comment:
This file will be included in the library, if it is in the main resources folder. I would prefer a dedicated test resources folder, which will be ignored while packaging the library.

Reply:
Fixed, now move to `test/resources`